openstack/dns/v2/recordsets/results.go

```diff
@@ -0,0 +1,113 @@
+package recordsets
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract interprets a GetResult, CreateResult or UpdateResult as a concrete RecordSet.
+// An error is returned if the original call or the extraction failed.
+func (r commonResult) Extract() (*RecordSet, error) {
+	var s *RecordSet
+	err := r.ExtractInto(&s)
+	return s, err
+}
+
+// GetResult is the deferred result of a Get call.
+type GetResult struct {
+	commonResult
+}
+
+// RecordSetPage is a single page of RecordSet results.
+type RecordSetPage struct {
+	pagination.LinkedPageBase
+}
+
+// IsEmpty returns true if the page contains no results.
+func (r RecordSetPage) IsEmpty() (bool, error) {
```

Comment:
I don't _think_ we need these anymore. At least, that was my intention when I wrote [this](https://github.com/gophercloud/gophercloud/blob/master/pagination/linked.go#L78). Let me know if something breaks if this is deleted.